src/Incidencia/ValidateIncidenciaForm.js

```diff
@@ -0,0 +1,22 @@
+export default function ValidateIncidenciaForm(form){
+
+    let errors = {}
+   
+    // Validación título
+    if(!form.titulo.trim()){
+        errors.titulo= "El título es un campo obligatorio" 
```

Comment:
Codacy has a fix for the issue: [Missing semicolon.](https://app.codacy.com/gh/park-inn-ispp/park-inn-frontend/pullRequest?prid=9179155)
```suggestion
        errors.titulo= "El título es un campo obligatorio"; 
```